apigw-http-api-lambda-ts-cdk/README.md

```diff
@@ -0,0 +1,70 @@
+# Amazon API Gateway HTTP API with AWS Lambda integration
+
+This pattern in CDK offers a boilerlate to generate an Amazon API Gateway HTTP API endpoint with "ANY" method from the specified path. The Lambda function provided in TypeScript only returns a dummy "HelloWorld".
+
+If Amazon API Gateway HTTP API (v2) covers all the features' requirements, it's recommended to use HTTP API (v2) over REST API (v1). The HTTP API has the advantage to be significantly less expensive, and it peforms better.
+
+Learn more about this pattern at Serverless Land Patterns: [https://serverlessland.com/patterns/apigw-http-api-lambda-ts-cdk](https://serverlessland.com/patterns/apigw-http-api-lambda-ts-cdk)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
```

Comment:
From my test, it looks like docker is also required. I would add it to the list.